app/store.ts

```diff
@@ -0,0 +1,47 @@
+import { configureStore, getDefaultMiddleware, Action } from '@reduxjs/toolkit';
+import { createHashHistory } from 'history';
+import { routerMiddleware } from 'connected-react-router';
+import { createLogger } from 'redux-logger';
+import { ThunkAction } from 'redux-thunk';
+// eslint-disable-next-line import/no-cycle
```

Comment:
I checked the same eslint configuration what we use here with the examples you provided and it _is_ screams about cyclic dependency there:/ I still thinks this is a shortcoming in eslint-plugin-import.

Reply:
Yeah, the lint plugin probably can't differentiate between an imported type and an actual value.